cumulusci/tasks/release_notes/task.py

```diff
@@ -119,16 +127,21 @@ class ParentPullRequestNotes(BaseGithubTask):
             "description": "force rebuilding of change notes from child branches in the given branch.",
             "required": False,
         },
+        "repos": {
```

Comment:
I'm fuzzier on the use case for this functionality in `ParentPullRequestNotes`. What does it do or is it expected to do?